adapters/turkiye.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * This code is responsible for implementing all methods related to fetching
+ * and returning data for the Taiwanese data sources.
+ */
+ 'use strict';
+
+ import * as fs from 'fs';
+ import { REQUEST_TIMEOUT } from '../lib/constants';
+ import { default as baseRequest } from 'request';
+ import { DateTime } from 'luxon';
+ import { parallel } from 'async';
+ import { convertUnits } from '../lib/utils';
+ import {parse} from 'wellknown';
+ const request = baseRequest.defaults({ timeout: REQUEST_TIMEOUT });
+ 
+ exports.name = 'turkiye';
+ 
+ /**
+  * Fetches the data for a given source and returns an appropriate object
+  * @param {object} source A valid source object
+  * @param {function} cb A callback of the form cb(err, data)
+  */
+  
+ exports.fetchData = function (source, cb) {
+ 
+     const ALL_DATA =
```

Comment:
This should be coming from the source JSON. e.g. https://github.com/openaq/openaq-fetch/blob/master/adapters/airnow-http.js#L73

Reply:
so I can replace ALL_DATA with source.url?

Reply:
yeahj